apps/fishing-map/features/map/popups/ContextLayers.hooks.ts

```diff
@@ -48,12 +46,13 @@ export const useContextInteractions = () => {
         return
       }
       batch(() => {
-        dispatch(
-          setDownloadActivityGeometry({
-            geometry: feature.geometry,
-            name: feature.value || feature.title,
-          })
-        )
+        // TODO get geometry from API too
```

Comment:
This needs to be done before merging right?

Reply:
Yes, working on it now